subtest/docs/pck-description.md

```diff
@@ -0,0 +1,405 @@
+[![Build Status](https://travis-ci.org/SubstraFoundation/distributed-learning-contributivity.svg?branch=master)](https://travis-ci.org/SubstraFoundation/distributed-learning-contributivity)
```

Comment:
Ok so if I understand correctly the entire old README content is now in this `pck-description.md` file. What is the rationale for this? As you kept some bits in the old README, it means we'll need to maintain duplicates of the same texts?
Maybe I just wonder about that because I am not familiar with the use of this `pck-description.md` file :smiley:

Reply:
Initially the pck_description was the old README, but as the README has changed, I keep the old description in this pck-description. It is maybe not really useful